Code/Editor/EditorViewportWidget.cpp

```diff
@@ -474,7 +474,7 @@ void EditorViewportWidget::Update()
             if (debugDisplay)
             {
                 const AZ::u32 prevState = debugDisplay->GetState();
-                debugDisplay->SetState(AzFramework::e_Mode3D | AzFramework::e_AlphaBlended | AzFramework::e_FillModeSolid | AzFramework::e_CullModeBack | AzFramework::e_DepthWriteOn | AzFramework::e_DepthTestOn);
+                debugDisplay->SetState((AZ::u32)AzFramework::e_Mode3D | (AZ::u32)AzFramework::e_AlphaBlended | (AZ::u32)AzFramework::e_FillModeSolid | (AZ::u32)AzFramework::e_CullModeBack | (AZ::u32)AzFramework::e_DepthWriteOn | (AZ::u32)AzFramework::e_DepthTestOn);
```

Comment:
Please use C++ rather than C style casts. Options are `static_cast` or `to_underlying`, for example. Even better would be to use `AZ_DEFINE_ENUM_BITWISE_OPERATORS` on the enum. Also for the other cases like this in this PR where enums are used with bitwise operators.